src/py-opentimelineio/opentimelineio/core/__init__.py

```diff
@@ -32,6 +38,17 @@
     release_to_schema_version_map,
 )
 
+Metadata: TypeAlias = Union[Dict[str, 'MetadataValue'], AnyDictionary]
```

Comment:
For these to work, we'll have to ove them into a [stub file](https://peps.python.org/pep-0484/#stub-files), more specifically `opentimelineio._otio.pyi`. We'll have to also create stubs for our bindings, which can be automated using [pybind11-stubgen](https://github.com/sizmailov/pybind11-stubgen).